c7n/policy.py

```diff
@@ -207,6 +214,35 @@ def run(self, *args, **kw):
                 "ActionTime", time.time() - at, "Seconds", Scope="Policy")
             return resources
 
+    def get_logs(self, start, end):
+        log_source = self.policy.ctx.output
+        log_gen = ()
+        if self.policy.options.log_group is not None:
+            session = utils.local_session(self.policy.session_factory)
+            log_gen = log_entries_from_group(
+                session,
+                self.policy.options.log_group,
```

Comment:
This is done now. I went ahead and updated the LambdaManager incarnation as well.